code/classes/DMSDocumentCart.php

```diff
@@ -181,6 +193,22 @@ public function getReceiverInfo()
     }
 
     /**
+     * Returns the recipients in a Viewable format
+     *
+     * @return ArrayData|bool
+     */
+    public function getReceiverInfoNice()
+    {
+        $result = false;
+
+        if (is_array($this->getReceiverInfo())) {
+            $result = ArrayData::create($this->getReceiverInfo());
+        }
+
+        return $result;
```

Comment:
What do you think about using ternary here? `return is_array($this->getReceiverInfo()) ? ArrayData::create($this->getReceiverInfo()) : false;`